test/common/network/address_impl_test.cc

```diff
@@ -119,6 +120,22 @@ TEST(Ipv4CompatAddressImplSocktTest, SocketBindAndConnect) {
   }
 }
 
+TEST(Ipv4InstanceTest, SockaddrToString) {
+  // Test addresses from various RF 5735 reserved ranges
+  static const char* addresses[] = {"0.0.0.0",        "0.0.0.255",       "0.0.255.255",
+                                    "0.255.255.255",  "192.0.2.0",       "198.151.100.1",
+                                    "198.151.100.10", "198.151.100.100", "10.0.0.1",
+                                    "10.0.20.1",      "10.3.201.1"};
```

Comment:
can you include a 255 in the first word?  Let's make sure we have testcases where each segment is 0, 255, and something else.